### PR TITLE
Toyota: comments for LTA steering signals

### PR DESCRIPTION
**Pull request by zorrobyte**
Dashcam LTA route:
4aa5d874f367adbc|2022-02-20--22-59-18

“Stock” LTA OP
4aa5d874f367adbc|2022-02-21--18-22-18

"Broken" OP LTA route (would not steer):
4aa5d874f367adbc|2022-02-21--18-37-54

Dashcam with disablements (LTA OEM steer overrides)
4aa5d874f367adbc|2022-02-22--14-47-21

Dadcam mode TSS2.5 stock
f15e3c37c118e841|2022-02-18--13-58-56

### CODE REVIEW

**Comment by jyoung8607** [comment]
You should annotate using `CM_ SG_` entries at the bottom of the file, see the other existing entries for examples. That's the correct way to add per-signal comment strings. Then you'll start seeing the comments appear in Cabana and other tools, which otherwise may not be able to parse this.

**Comment by zorrobyte** [comment]
Will fix

**Comment by zorrobyte** [comment]
Should be good now

File: generator/toyota/toyota_nodsu_pt.dbc [hunk 1]
```diff
@@ -33,6 +33,13 @@ BO_ 1014 BSM: 8 XXX
  SG_ ADJACENT_ENABLED : 7|1@0+ (1,0) [0|1] "" XXX
  SG_ APPROACHING_ENABLED : 15|1@0+ (1,0) [0|1] "" XXX
 
+CM_ SG_ 401 PERCENTAGE "driver override percentage (0-100), very close to steeringPressed in OP";
+CM_ SG_ 401 SETME_X64 "ramps to 0 smoothly then back on falling edge of STEER_REQUEST if BIT isn't 1";
+CM_ SG_ 401 ANGLE "angle of car relative to lane center on LTA camera";
+CM_ SG_ 401 STEER_ANGLE_CMD "desired angle, OEM steers up to 95 degrees, no angle limit but torque will bottom out";
+CM_ SG_ 401 STEER_REQUEST "enable bit for steering, 1 to steer, 0 to not";
+CM_ SG_ 401 BIT "has coorelation to STEER_REQUEST";
```
Comment (adeebshihadeh) [inline generator/toyota/toyota_nodsu_pt.dbc:41]:
```suggestion
CM_ SG_ 401 BIT "has correlation to STEER_REQUEST";
```

**Pull request merged**